chimney/src/main/scala/io/scalaland/chimney/internal/macros/MappingMacros.scala

```diff
@@ -145,16 +152,27 @@ trait MappingMacros extends Model with TransformerConfiguration {
       lookupName: String,
       wasRenamed: Boolean,
       From: Type
-  )(ms: MethodSymbol): Boolean = {
+  )(ms: MethodSymbol): AccessorResolution = {
     val sourceName = ms.name.decodedName.toString
     if (config.enableBeanGetters) {
       val lookupNameCapitalized = lookupName.capitalize
-      sourceName == lookupName ||
-      sourceName == s"get$lookupNameCapitalized" ||
-      (sourceName == s"is$lookupNameCapitalized" && ms.resultTypeIn(From) == typeOf[Boolean])
+      if (sourceName == lookupName ||
+          sourceName == s"get$lookupNameCapitalized" ||
+          (sourceName == s"is$lookupNameCapitalized" && ms.resultTypeIn(From) == typeOf[Boolean])) {
+        AccessorResolution.Resolved(ms, wasRenamed)
```

Comment:
We don't yet support renaming java beans getters. This change is not sufficient in order to provide such support. And I think we can safely set this flag to `false` to better communicate the intent.

Reply:
👌